src/cmd/cmd_controller.js

```diff
@@ -634,16 +635,20 @@ class EmbarkController {
         engine.init({}, callback);
       },
       function startServices(callback) {
+
         engine.startService("processManager");
         engine.startService("libraryManager");
-        engine.startService("web3", {wait: true});
+        engine.startService("codeRunner");
+        engine.startService("web3", {wait: true, node: options.node});
```

Comment:
Why do you reverse them?

Reply:
Copied the services used by the console as it's the console service that makes EmbarkJS available (does it make sense, i don't know).
The order probably doesn't make much difference, I just went the fast way of making sure to have the same services.